taxcalc/cli/tc.py

```diff
@@ -38,12 +44,12 @@ def main():
     parser.add_argument('--reform',
                         help=('REFORM is name of optional JSON reform file. '
                               'No --reform implies use of current-law '
-                              'policy.'),
+                              'policy (clp).'),
                         default=None)
     parser.add_argument('--assump',
                         help=('ASSUMP is name of optional JSON economic '
-                              'assumption file.  No --assump implies use of '
-                              'static analysis assumptions.'),
+                              'assumptions file.  No --assump implies use of '
+                              'standard (std) analysis assumptions.'),
```

Comment:
I think "base" might be more appropriate here than "standard" as the word "standard" has normative implications.